WordPress/Classes/Utility/Keychain.swift

```diff
@@ -0,0 +1,45 @@
+@objcMembers
+class Keychain {
+
+    static let shared = Keychain()
+
+    private let useSharedKeychain: () -> Bool
```

Comment:
Wouldn't `shouldUseSharedKeychain` be a better name? Current one sounds like it will perform a change on the state rather than a bool. Sorry I just noticed this while trying to suggest a name for the class

Reply:
No worries and yeah, that is a better name. It's always tough balancing how long the name is versus a good name. I can make that change with the class name change.